fe/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -262,6 +262,11 @@ public boolean isCountDistinctBitmapOrHLL() {
             return false;
         }
 
+        // if expr isn't analyzed, the type is INVALID
+        if (!getChild(0).isAnalyzed) {
+            getChild(0).analyzeNoThrow(analyzer);
```

Comment:
I'm not sure if it is OK to analyze it here.
Is it possible to call this function after analyzed?

Reply:
I also considered this problem，But I think wrong before.

Done.